commands/bmc.vegman

```diff
@@ -363,24 +363,34 @@ function cmd_remoteimage {
 # @sudo cmd_remoteimage_connect admin
 # @doc cmd_remoteimage_connect
 # Connect to the remote server via the specified protocol
-#   --nbd <0..10>     use NBD protocol to connect the remote server with specified NBD device number
-#   address[:port]    remote server address
+#   --nbd <0..10>         use NBD protocol to connect the remote server with specified NBD device number
+#   -e,--export name      the export name from the NBD server
```

Comment:
What is "export name"? Can the comment be a bit more detailed?

Reply:
The "export name" is an idiom of NBD server specific. This option means the literal short name of exporting the remote file, which is clear for the NBD-server consumers. 
I don't think that we should provide detailed information about each NBD-client argument, which might be got from 'man nbd-server|nbd-client.'

Reply:
But I believe the exported name should be an optional argument.
I will remove the test that this option is present.

Reply:
From nbd-client man:
```
-N | --name -- Specifies the name of the export that we want to use.
 If not specified, nbd-client will ask for a "default" export, if one exists on the server.
```
Well, I can't found any additions to the comment to make it more detailed.